modules/auxiliary/server/mysql_rogue.py

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/env python3
+
+# mysql rogue server requirements
+import socket
+import asyncore
+import asynchat
+import struct
+import logging
+import logging.handlers
+import codecs
+from argparse import ArgumentParser
+from enum import Enum
+import metasploit.module as module
+
+metadata = {
+    'name': 'mysqlrogue',
+    'description': '''
+        The script starts a MySQL server that requests and retrieves files from clients that connect to it.
+    ''',
+    'authors': [
+        'Moritz Nentwig'
```

Comment:
This should also credit Gifts for the original rogue server:

https://github.com/Gifts/Rogue-MySql-Server

```suggestion
        'Moritz Nentwig', # Metasploit
        'Gifts' # Rogue MySQL server
```